sync-server/Cargo.toml

```diff
@@ -20,5 +21,5 @@ env_logger = "^0.8.3"
 rusqlite = { version = "0.25", features = ["bundled"] }
 
 [dev-dependencies]
-actix-rt = "^2.2.0"
+actix-rt = "^1.1.1"
```

Comment:
Is there a reason to use the older release of actix-rt?

Reply:
Oddly, actix-web depends on actix-rt-1.1.1.  Using a newer (incompatible) actix-rt in tests leads to errors about the system not being started (since they use two entirely different System types).

Reply:
Ah, fair enough!

Irrelevant to this PR but I've started playing around with [axum](https://github.com/tokio-rs/axum) for a side-project, which seems very promising. Although it's another "v0.2 web framework", it seems to basically just be assembling various parts of the tokio-rs ecosystem, and those parts are quite well established, hyper etc